fashion-messaging-app-ui/src/App.jsx

```diff
@@ -34,10 +34,9 @@ useEffect(() => {
   clientAPI.photos
     .search({ query, per_page: PHOTOS })
     .then(async response => { 
-      setLoading(false);
 
       var myHeaders = new Headers();
-      myHeaders.append("x-api-key", "1a4092c4563cc701627b1616fa6053190c6b538fd82b828d3fe20416bcc3dd3e");
+      myHeaders.append("x-api-key", "4d552e9f30522b1dec7c712f83c67c235be86e25ec14b4ba3493383ec7b81d3f");
 
```

Comment:
I changed this because I reached API key rate limit so I signed up with another email to get a new one. This will be better to manage once I have implemented caching